evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -110,6 +110,7 @@ def __init__(
         self.verbose = verbose
         self._selected_cat_cols = []
         self._split = False
+        self._ensembling = True if not is_time_series(self.problem_type) else False
```

Comment:
I can add ensembling as a parameter to `DefaultAlgorithm` as well but currently I don't believe we have a use case for it.